databases/week1/sqlhomeworkweek1.sql

```diff
@@ -0,0 +1,47 @@
+-- 1. Find out how many tasks are in the task table
+SELECT COUNT(id) AS task FROM task;
+
+-- 2. Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(id) FROM task 
+WHERE due_date IS NULL;
+
+-- 3. Find all the tasks that are marked as done
+SELECT * FROM task WHERE status_id = 3;
+
+-- 4. Find all the tasks that are not marked as done
+SELECT * FROM task WHERE status_id = 0;
```

Comment:
This one gives no rows in the result. It should give you 23 rows when correct. Can you spot the mistake? Hint: The WHERE clause should be almost identical to the query above